scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hvssop.yaml

```diff
@@ -0,0 +1,55 @@
+FileHeader:
+  library_Suffix: 'SO'
+  device_type: 'HVSSOP'
+
+HVSSOP-10-1EP_3x3mm_P0.5mm_EP1.57x1.88mm:
+  size_source: 'https://www.ti.com/lit/ds/symlink/bq24090.pdf'
+
+  body_size_x:
+    nominal: 3
+    tolerance: 0.100
+  body_size_y:
+    nominal: 3
+    tolerance: 0.100
+  overall_height:
+    maximum: 1.1
+
+  overall_size_x:
+    nominal: 4.9
+    tolerance: 0.152
+  lead_width:
+    minimum: 0.17
+    maximum: 0.27
+  lead_len:
+    nominal: 0.53
+    tolerance: 0.152
+
+  pitch: 0.5
+  num_pins_x: 0
+  num_pins_y: 5
+
+  EP_size_x:
+    nominal: 1.57
+    tolerance: 0
```

Comment:
please remove the tolerance:0 line
Also remove the un-commented lines while you are at it.